public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
-  return 'Hello, World!';
+function buildProduct(product) {
```

Comment:
Honestly. I am not a religious person. I don't believe in prophets, commandments
or rules XD... So. What kind of value will add that change to the current code/file?